wurst/systems/entities/bosses/BossUtilities.wurst

```diff
@@ -1,46 +1,54 @@
 package BossUtilities
 
+// Standard-library imports:
 import Assets
-import GroupUtils
 import ClosureTimers
+
+// Local imports:
 import HealingSystem
+import GroupUtils
+import UnitEntity
 
-public function checkInvulnerability(unit mini_Boss, region mini_Boss_Region) returns bool
+public function checkInvulnerability(region miniBossRegion, unit miniBoss) returns bool
```

Comment:
I don't think that there's really a distinction when it comes to miniboss or a reason why this system would be limited to that. I would omit the `mini` from all names, although implying that this is only for bosses should be fine.